frappe/hooks.py

```diff
@@ -169,7 +170,8 @@
 		"on_trash": [
 			"frappe.desk.notifications.clear_doctype_notifications",
 			"frappe.workflow.doctype.workflow_action.workflow_action.process_workflow_actions",
-			"frappe.event_streaming.doctype.event_update_log.event_update_log.notify_consumers"
+			"frappe.event_streaming.doctype.event_update_log.event_update_log.notify_consumers",
+			"frappe.custom.doctype.customize_form.customize_form.genrate_qr_code_file"
```

Comment:
You should use delete_qr_code_file instead of genrate_qr_code_file in on_trash

Reply:
Thanks. It is resolved